### ISSUE TITLE
DISCLOSURE OF A MAJOR BUG IN CRYPTONOTE BASED CURRENCIES

### ISSUE THREAD

**Issue by stefek99** [cryptonotefoundation/cryptonote#39]
See blog: https://getmonero.org/2017/05/17/disclosure-of-a-major-bug-in-cryptonote-based-currencies.html

Here is the pull request: https://github.com/monero-project/monero/pull/1744#issuecomment-306456536

Just checking...

Looking by the commit dates fix hasn't been applied yet?

**Comment by valiant1x** [comment]
I don't believe the bug affects this implementation. In the Monero and AEON projects, a function comparing the domain image was modified (`check_tx_inputs_keyimages_domain`). This repo has no such function. IIRC the Monero and AEON versions are from ~2014, this repo appears to be from closer to 2015.

**Comment by SentRaise** [comment]
@valiant1x `check_tx_inputs_keyimages_domain` **is** the fix for the issue.

Since this repo does not have that function, it is vulnerable.

**Comment by valiant1x** [comment]
@SentRaise You're exactly right, I was 100% incorrect. I am not sure where my head was at when I posted that comment..
